.github/workflows/ci.yaml

```diff
@@ -59,6 +59,26 @@ jobs:
         run: |
           make unit-tests
 
+  int-test:
```

Comment:
if you add something here don't forget to add it to the 'validate success' at the bottom

Reply:
updated. thanks